Library/.rubocop.yml

```diff
@@ -189,8 +189,25 @@ Style/TrailingCommaInArguments:
   EnforcedStyleForMultiline: comma
 
 # we have too many variables like sha256 where this harms readability
-Style/VariableNumber:
+Naming/VariableNumber:
   Enabled: false
 
 Style/WordArray:
   MinSize: 4
+
+# we want to add this slowly and manually
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+# generally rescuing StandardError is fine
+Lint/RescueWithoutErrorClass:
+  Enabled: false
+
+# implicitly allow EOS as we use it everywhere
```

Comment:
Thanks for catching this. My `$EDITOR`'s Rubocop plugin has already been driving me up to the wall over `EOS` stuff.

Reply:
You can thank me for the fit I pitched about it lol

Reply:
I don't understand what the issue is with `EOS` to be honest. It'd seem silly to get ultra-specific inside formulae at least.

Reply:
https://github.com/bbatsov/rubocop/pull/4467

I get the sense that whatever conversations there were about this didn't really occur in the open.

Reply:
One nice thing is that in TextMate you get syntax highlighting inside heredocs if you use e.g. `RUBY`, but if it's just text, I'm also in favour of `EOS`.